### DISCUSSION TITLE
ping: Lacking privilege for icmp socket (--net=)

### DISCUSSION THREAD

**Discussion by Abdalnablse10** [netblue30/firejail#5769] [Q&A] [answered]
'firejail ping google.com' works
But for some reason when adding --net=eth0 it gives me this ' ping: Lacking privilege for icmp socket. '
What should I do to give it privilege for icmp socket?

**Comment by rusty-snake** [comment, marked as answer]
Closing in favour of ~~#5774~~:

* #5775
